rosidl_generator_rs/package.xml

```diff
@@ -13,7 +13,8 @@
   <buildtool_depend>rosidl_runtime_rs</buildtool_depend>
 
   <buildtool_export_depend>ament_cmake</buildtool_export_depend>
-  <buildtool_export_depend>rosidl_cmake</buildtool_export_depend>
+  <buildtool_export_depend condition="humble >= $ROS_DISTRO">rosidl_cmake</buildtool_export_depend>
+  <buildtool_export_depend condition="$ROS_DISTRO > humble">rosidl_pycommon</buildtool_export_depend>
   <buildtool_export_depend>rosidl_runtime_rs</buildtool_export_depend>
   <buildtool_export_depend>rosidl_typesupport_c</buildtool_export_depend>
   <buildtool_export_depend>rosidl_typesupport_interface</buildtool_export_depend>
```

Comment:
Please add `buildtool_depend` and `buildtool_export_depend` for `ros_environment`, which provides access to the `ROS_DISTRO` environment variable. Although usually when building locally `ROS_DISTRO` is available, it's not guaranteed unless `ros_environment` is added as a dependency.

Reply:
Done